decidim-core/app/packs/src/decidim/sw/push-permissions.js

```diff
@@ -0,0 +1,83 @@
+window.addEventListener("DOMContentLoaded", async () => {
+  if ("serviceWorker" in navigator) {
+    const toggle = document.getElementById("allow_push_notifications")
+
+    if (toggle) {
+      const reminder = document.querySelector(".push-notifications__reminder")
```

Comment:
Do not use specific CSS classes to point them from any javascript resources. You can:
- Add a unique `id`
- Use a hacky CSS class like `.js-whatever`
- Use `data-attributes`